src/Functions/Lemmatizers.cpp

```diff
@@ -0,0 +1,93 @@
+#include <Common/Exception.h>
```

Comment:
Maybe place all these functions in a separate subdirectory? `Functions/NLP`

Reply:
I actually moved Lemmatizers and SynonymsExtension .h and .cpp files to `Interpreters` directory, because odbc-bridge build could see functions declaration in Lemmatizers.h and SynonymsExtension.h (they are included in Context.cpp), but .cpp files with actual code were in `clickhouse_functions` library, so linker would say something like `undefined symbol DB::Lemmatizers::Lemmatizers()`